.github/workflows/lint-css.yml

```diff
@@ -0,0 +1,48 @@
+name: CSS Linting
+on:
+  push:
+    branches-ignore:
+      - "dependabot/**"
+    paths:
+      - "package*.json"
+      - "Gruntfile.coffee"
+      - "stylelint.config.js"
+      - ".sass-lint.yml"
```

Comment:
This and the step below should be removed if https://github.com/wet-boew/GCWeb/pull/2523 lands before this one